docs/self-managed/platform-deployment/helm-kubernetes/guides/aws-marketplace.md

```diff
@@ -0,0 +1,494 @@
+---
+id: aws-marketplace
+title: "Install AWS Marketplace"
+description: "Learn how to Install the AWS Marketplace."
+---
+
+<!--- Can you give us a 2- or 3-sentence overview of the guide here and why a user would want to install this? --->
```

Comment:
See comment here.

Reply:
that's a good suggestion. though I'm not sure I have a good answer for that.

Reply:
@felix-mueller @theburi ,

How would you answer this ?

Reply:
I took a blob Felix suggested and added it to the PR
> You can subscribe to Camunda directly from the AWS Marketplace. You then have the convenience of viewing your Camunda subscription as part of your AWS bill, and you do not have to supply any additional billing information to Camunda.

Reply:
Rockstar 🎸